src/common/grpc-expr/src/error.rs

```diff
@@ -104,6 +105,25 @@ pub enum Error {
         #[snafu(implicit)]
         location: Location,
     },
+
+    #[snafu(display("Unknown proto column datatype: {}", datatype))]
+    UnknownColumnDataType {
+        datatype: i32,
+        #[snafu(implicit)]
+        location: Location,
+        #[snafu(source)]
+        error: prost::DecodeError,
+    },
+
+    #[snafu(display(
+        "Full-text index only supports string type, column: {column_name}, unexpected type: {column_type:?}"
```

Comment:
To maintain consistency.

```suggestion
        "Fulltext index only supports string type, column: {column_name}, unexpected type: {column_type:?}"
```